InteractionBase/src/handlers/HViewItem.cpp

```diff
@@ -42,6 +42,27 @@ void HViewItem::keyPressEvent(Visualization::Item *target, QKeyEvent *event)
 {
 	if (event->modifiers() == Qt::NoModifier && event->key() == Qt::Key_Return)
 		GenericHandler::showCommandPrompt(target, "add ");
+	else if (target->focusedChild() && event->modifiers() == Qt::ControlModifier
+		&& (event->key() == Qt::Key_I || event->key() == Qt::Key_J
+			|| event->key() == Qt::Key_K || event->key() == Qt::Key_L))
+	{
+		auto child = target->focusedChild();
+		//TODO@cyril For some reason it does not work always without this
```

Comment:
Hmm, we had this before, but eventually removed it. I'll merge this, and will look into it.